tests/mysql-8.0.42.sh

```
@@ -0,0 +1,50 @@
+type=mysql
+version=8.0.42
+
+cd ../$type/..
+
+date=$(date +%Y%m%d%H%M%S)
+md5="md5"
+[ "$(getOS)" = "linux" ] && md5="md5sum"
+hash=$(echo "dbdb-$date" | $md5 | cut -d ' ' -f 1)
+
+echo "# Test create"
+./$type/create.sh dbdb-test-$hash $version random
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)